ui/puzzle/src/interfaces.ts

```diff
@@ -155,7 +163,7 @@ export interface PuzzleReplay {
   days: number;
 }
 
-export interface PuzzleGame {
+export interface PuzzleGame extends KeyboardMoveRootGame {
```

Comment:
Explicitly implementing this interface here solved a typing issue I was encountering related to weak types (interfaces with only optional fields)